src/ingest-pipeline/misc/tools/split_and_create.py

```diff
@@ -263,12 +264,24 @@ def populate(row, source_entity, entity_factory, dryrun=False, components=None):
     if dryrun:
         print(f"copy {old_contrib_path} to {extras_path}")
     else:
-        copy2(source_entity.full_path / old_contrib_path, extras_path)
+        src_path = source_entity.full_path / old_contrib_path
+        if src_path.exists():
+            copy2(src_path, extras_path)
+        else:
+            moved_path = kid_path / new_contrib_path
+            print(f"""Probably already copied/moved {src_path} 
+                      to {moved_path} {"it exists" if moved_path.exists() else "missing file"}""")
```

Comment:
Validation should catch if any referenced contributors_paths don't actually exist, so that is a safe assumption, but if there's not a contributors.tsv at either src_path or moved_path is that a problem?

Reply:
i.e. the "missing file" message case.

Reply:
That's a good question, I don't remember if the file is mandatory on any directory schemas on IVT but ingest-pipeline does rely on it to get information to build metadata for the entity. So, if no contributors.tsv reorganization will fail.